src/theme/src/components/inputs.js

```diff
@@ -0,0 +1,85 @@
+import { minorScale } from '../../../scales'
+
+export default function getInputStyles(theme) {
+  const { tokens } = theme
+
+  return {
+    baseStyle: {
+      WebkitAppearance: 'none',
+      MozAppearance: 'none',
+      lineHeight: '12px',
+      border: 'none',
+      borderRadius: '4px',
+      fontFamily: tokens.fontFamilies.ui
+    },
+
+    appearances: {
+      default: {
+        backgroundColor: 'white',
+        border: `1px solid ${tokens.colors.gray400}`,
+
+        _invalid: {
+          borderColor: tokens.colors.red500
+        },
+
+        _focus: {
+          outline: 'none',
+          transition: 'box-shadow 80ms ease-in-out',
+          border: `1px solid ${tokens.colors.blue200}`,
+          boxShadow: `0 0 0 2px ${tokens.colors.blue100}`
+        },
+
+        _disabled: {
+          cursor: 'not-allowed',
+          backgroundColor: tokens.colors.gray100
+        },
+
+        _placeholder: {
+          color: tokens.colors.gray600
+        },
+
+        _placeholderHover: {
+          color: tokens.colors.gray700
+        }
+      },
+      none: {
```

Comment:
What's the difference between default vs none?

Reply:
I think the `none` appearance should be `transparent` for the background! But otherwise it's meant to be one of those "floating" inputs (we use this in `<TagInput />` for example)

Reply:
default is the bordered visual, none has no borders